package.json

```diff
@@ -10,6 +10,7 @@
   "devDependencies": {
     "@percy/cypress": "^1.0.2",
     "cypress": "^3.1.5",
-    "http-server": "^0.11.1"
+    "http-server": "^0.11.1",
+    "https-server": "^0.1.2"
```

Comment:
Do you ever use this `https-server` package? Might not be needed.

Reply:
Very good catch! I did not intend to add that 😱